openmdao/vectors/vector.py

```diff
@@ -558,7 +560,9 @@ def set_var(self, name, val, idxs=_full_slice, flat=False):
             raise ValueError(f"{self._system().msginfo}: Attempt to set value of '{name}' in "
                              f"{self._kind} vector when it is read only.")
 
-        if not isinstance(idxs, Indexer):
+        if idxs == _full_slice:
```

Comment:
I would use `if idxs is _full_slice` here.

Reply:
Fixed